docs/src/man/basics.md

```diff
@@ -0,0 +1,1220 @@
+# First Steps with DataFrames
+
+The first and imporatant part to work with DataFrames.jl, is to install the `DataFrames.jl`
+package which is available thorugh the Julia package system and can be installed using the
+following commands:
+
+```julia
+julia> using Pkg
+julia> Pkg.add("DataFrames")
+```
+
+Or,
+
+```julia
+julia> ]
+
+julia> add DataFrames
+```
+
+To make sure everything works as expected, try to load the package and if you have the time execute
+its test suits:
+```julia
+julia> using DataFrames
+julia> using Pkg
+julia> Pkg.test("DataFrames")
+```
+
+Throughout the rest of the tutorial we will assume that you have installed the DataFrames package and
+have already typed `using DataFrames` to bring all of the relevant variables into your current namespace.
+Let's get started by loading the DataFrames package:
+```julia
+julia> using DataFrames
+```
+
+The object of the `DataFrame` type represent a data table as a series of vectors, each correspondng to a
```

Comment:
this sentence might be misleading. it is rather better to say something that each row is an observation and each column is a feature or something similar. The reason is that `DataFrame` IS NOT a collection of its columns.